requirements.txt

```diff
@@ -1,3 +1,4 @@
-ops
+git+https://github.com/balbirthomas/operator/@provider-consumer-lib#egg=ops
```

Comment:
I would use `git+https://github.com/canonical/operator/@master#egg=ops` since AFAIK your PR is already merged.

Reply:
Ack will do.